conans/client/packager.py

```diff
@@ -41,11 +41,13 @@ def export_pkg(conanfile, package_id, src_package_folder, package_folder, hook_m
     return prev
 
 
-def create_package(conanfile, package_id, source_folder, build_folder, package_folder,
-                   install_folder, hook_manager, conanfile_path, ref, local=False,
-                   copy_info=False):
-    """ copies built artifacts, libs, headers, data, etc. from build_folder to
-    package folder
+def package_method_run(conanfile, package_id, source_folder, build_folder, package_folder,
```

Comment:
better `packager.run_package_method()`? `packager.package()`?

Reply:
Definitely NOT packager.package(). 
The whole purpose of the rename is to clearly identify what it is doing, be able to grep it. ``package()`` is used in codebase for 100 different things. Renamed to ``run_package_method()``